assets/app/views/create/fromimage.html

```diff
@@ -52,7 +60,7 @@
 
                 <div class="form-group">
                   <label for="sourceUrl" class="required">Git Repository URL</label>
-                  <div ng-class="{'has-warning': form.sourceUrl.$touched && !sourceURLPattern.test(buildConfig.sourceUrl), 'has-error': (form.sourceUrl.$error.required && form.sourceUrl.$touched)}">
+                  <div ng-class="{'has-warning': form.sourceUrl.$touched && !sourceURLPattern.test(buildConfig.sourceUrl), 'has-error': (form.sourceUrl.$error.required && form.sourceUrl.$dirty)}">
```

Comment:
Yes, that's the behavior we've had, and I haven't changed it. I can switch it to $dirty. You're usually pasting into this field, not typing.